src/kosha/db/kriti.clj

```diff
@@ -0,0 +1,21 @@
+(ns kosha.db.kriti
+  (:require [kosha.db.util :as db-util]))
+
+(defn kriti
+  "Retrieve details of a kriti given it's kriti_id."
+  [kriti-id]
+  (let [q ["SELECT k.kriti_id, k.name, k.composer, k.taala, k.url, k.lyrics, k.meaning, k.ragam_id, k.data_source, r.name AS ragam_name
```

Comment:
Because:
a) I find it helpful to be able to look at the query within the code and see what fields are relevant, instead of having to look in the db or where the returned fields are being used.
b) I've read that `*` is an SQL anti-pattern, although the reasons for that are probably not relevant to this specific case.

Do you think cleaning up the query string here is more valuable than a) ?